src/dataflow/src/render/mod.rs

```diff
@@ -1131,7 +1131,7 @@ where
 
                 MirRelationExpr::Reduce { input, .. } => {
                     self.ensure_rendered(input, scope, worker_index);
-                    self.render_reduce(relation_expr, scope);
+                    self.render_reduce(relation_expr);
```

Comment:
Is this important? I suspect we'll want to introduce scopes in this implementation (or around it).

Reply:
this is not important. i just stopped using the passed in scope. we can add it back when needed?